config/seata-config-spring-cloud/src/main/java/io/seata/config/springcloud/SpringCloudConfiguration.java

```diff
@@ -0,0 +1,84 @@
+/*
+ *  Copyright 1999-2019 Seata.io Group.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package io.seata.config.springcloud;
+
+import io.seata.common.util.StringUtils;
+import io.seata.config.AbstractConfiguration;
+import io.seata.config.ConfigChangeListener;
+
+import java.util.List;
+
+
+public class SpringCloudConfiguration extends AbstractConfiguration<ConfigChangeListener> {
+
+    private static final String CONFIG_TYPE = "SpringCloudConfig";
+    private static SpringCloudConfiguration instance;
+    private static final String PREFIX = "seata.";
+    public static SpringCloudConfiguration getInstance() {
```

Comment:
Please add a private constructor without parameters to avoid it being instantiated.